src/category/liftable.lean

```diff
@@ -0,0 +1,150 @@
+
+import data.equiv.basic
+import tactic
+
+universes u₀ v₀ u₁ v₁ v₂ w w₀ w₁
+
+class liftable (α : Type u₀) (β : (Type v₀)) :=
+  (lift {}   : α ≃ β)
+
+-- instance {α} : liftable α (ulift α) :=
```

Comment:
OH NOES (is this meant to be commented)

Reply:
That looks like a useful instance. Let's see if it breaks something.